monkey/monkey_island/cc/resources/agent_logs.py

```diff
@@ -0,0 +1,27 @@
+from http import HTTPStatus
+
+from flask import request
+
+from monkey_island.cc.repository import IAgentLogRepository
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+
+
+class AgentLogs(AbstractResource):
+    urls = ["/api/agent-logs/<string:agent_id>"]
```

Comment:
There are mypy errors on 18 and 25 because the agent log repository expects a UUID, not a string. See https://flask.palletsprojects.com/en/2.2.x/quickstart/#variable-rules